README.md

```diff
@@ -6,9 +6,9 @@ Below are the instructions on how to install, configure, and generate a Rubrik C
 ## Pre-requisites
 The following PowerShell modules are required for generating a Rubrik As Built report.
 
-Each of these modules can be easily downloaded and installed via the PowerShell Gallery 
+Each of these modules can be easily downloaded and installed via the PowerShell Gallery
 
-- [Rubrik CDM Powershell SDK](https://www.powershellgallery.com/packages/Rubrik/) 
+- [Rubrik CDM Powershell SDK](https://www.powershellgallery.com/packages/Rubrik/)
     *Note:* The Rubrik As Built Report Module currently relies on the `devel` branch of the Rubrik PowerShell SDK. Follow the instructions below to ensure the `devel` branch is the active installation.
```

Comment:
Is this still required now that the module is publish to the PS Gallery?

Reply:
No - updated docs to show that